test/index.js

```diff
@@ -335,6 +335,11 @@ t.test('prompt, accepts', async t => {
   const npxCache = resolve(testdir, 'npxCache')
   const libexec = t.mock('../lib/index.js', {
     '@npmcli/ci-detect': () => false,
+    'proc-log': {
```

Comment:
Is there a test now that has a log without this method?

Reply:
Good suggestion. I'll take it. Thank you!